app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -506,6 +507,9 @@ class BrowserFragment :
 
             DownloadState.Status.COMPLETED -> {
                 Downloads.downloadCompleted.record(NoExtras())
+
+                TelemetryWrapper.downloadDialogDownloadEvent(sentToDownload = true)
```

Comment:
self note: we always send true only here?

Reply:
Yes, we send `false` when the download is canceled: https://github.com/mozilla-mobile/focus-android/blob/b34f1ebf56eb6dbb9f1ef214a994c7fca87fc72d/app/src/main/java/org/mozilla/focus/telemetry/TelemetryMiddleware.kt#L61-L66